Fix docs to make property listing in the website docs more useful. (#5746)
chef/inspec: docs-chef-io/content/inspec/resources/postgres_conf.md

- `('path')` is the non-default path to the `postgresql.conf` file (optional)
- `should eq 'value'` is the value that is expected

## Properties

> This resource supports any of the settings listed in an postgresql.conf file as properties for e.g. max_connections

Reviewer: ```suggestion
This resource supports any of the settings listed in a `postgresql.conf` file as properties. For example, `max_connections`:
```
